metadata-ingestion/src/datahub/ingestion/source/mode.py

```diff
@@ -560,16 +561,16 @@ def _get_chart_type(self, token: str, display_type: str) -> Optional[str]:
         }
         if not display_type:
             self.report.report_warning(
-                key="mode-chart-type-mapper",
-                reason=f"{token}: Chart type is missing. Setting to None",
+                title="Unrecognized chart type found",
+                message=f"{token}: Chart type is missing. Setting to None",
             )
             return None
         try:
             chart_type = type_mapping[display_type]
         except KeyError:
             self.report.report_warning(
-                key="mode-chart-type-mapper",
-                reason=f"{token}: Chart type {display_type} not supported. Setting to None",
+                title="Unrecognized chart type found",
+                message=f"{token}: Chart type {display_type} not supported. Setting to None",
```

Comment:
we should probably start demoting some of these to info logs

Reply:
Actually yes I'd agree

Reply:
Also cleaning up the message to avoid embedding whatever this token is.